docs/build/html/S3Storage.html

```diff
@@ -0,0 +1,296 @@
+<span class="p">)</span>
+</pre></div>
+</div>
+</div></blockquote>
+</div>
+<div class="section" id="automatic-transfers-to-from-sts-storage-locations-using-boto3-with-synapseclient">
+<h3>Automatic transfers to/from STS storage locations using boto3 with synapseclient<a class="headerlink" href="#automatic-transfers-to-from-sts-storage-locations-using-boto3-with-synapseclient" title="Permalink to this headline">¶</a></h3>
+<p>The Python Synapse client can be configured to automatically use STS tokens to perform uploads and downloads to enabled
+storage locations using an installed boto3 library rather than through the traditional Synapse client APIs.
+This can improve performance in certain situations, particularly uploads of large files, as the data transfer itself
+can be conducted purely against the AWS S3 APIs, only invoking the Synapse APIs to retrieve the necessary token and
+to update Synapse meta data in the case of an upload. Once configured to do so, retrieval of STS tokens for supported
+operations occurs automatically without any change in synapseclient usage.</p>
+<p>To enable STS/boto3 transfers, do the following:</p>
+<ol class="arabic simple">
+<li><p>Ensure that boto3 is installed in the same Python installation as synapseclient.</p></li>
+</ol>
+<blockquote>
+<div><div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">pip</span> <span class="n">install</span> <span class="n">boto3</span>
+</pre></div>
+</div>
+</div></blockquote>
+<ol class="arabic simple" start="2">
+<li><p>Update your Synapse client configuration file (typically “.synapseConfig” in your $HOME directory unless otherwise
+configured) with the following (adding the [transfer] section if it is not already present), or configure
+it on a per Synapse client object basis by setting the <strong>use_boto_sts_transfers</strong> property.</p></li>
+</ol>
+<blockquote>
+<div><div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="c1"># add to .synapseConfig to automatically apply as default for all synapse client instances</span>
+<span class="p">[</span><span class="n">transfer</span><span class="p">]</span>
+<span class="n">use_boto_sts</span><span class="o">=</span><span class="n">true</span>
+
+<span class="c1"># alternatively set on a per instance basis within python code</span>
+<span class="n">syn</span><span class="o">.</span><span class="n">use_boto_sts_transfers</span> <span class="o">=</span> <span class="kc">True</span>
+</pre></div>
+</div>
+</div></blockquote>
+<p>Once this is done any get or store operation through the configured client to or from STS enabed storage locations
+will be automatically conducted using the boto3 library rather than through the normal Synapse APIs.
+Note that if boto3 is not installed then these settings will have no effect.</p>
```

Comment:
```suggestion
<p>Note that if boto3 is not installed, then these settings will have no effect.</p>
```
I removed the leading sentence as I think it is well-stated above.